src/main/starlark/builtins_bzl/common/cc/cc_compilation_helper.bzl

```diff
@@ -130,11 +131,10 @@ def _compute_public_headers(
         module_map_headers.append(original_header)
 
     virtual_headers = module_map_headers + non_module_map_headers
-
     return struct(
         headers = virtual_headers,
         module_map_headers = module_map_headers,
-        virtual_include_path = cc_internal.bin_or_genfiles_relative_to_unique_directory(actions = actions, unique_directory = _VIRTUAL_INCLUDES_DIR),
```

Comment:
I thought this was still necessary because virtual includes are still under genfiles at Google, but apparently not (verified both by reading the code and ascertaining that `outputs_to_bindir` is set to true by default and by checking where `_virtual_includes` directories go). So then this change seems to be an obvious improvement to me because this code probably has to match the way `virtual_include_dir` is computed above.